crates/rb-sys/src/stable_api/ruby_2_6.rs

```diff
@@ -88,6 +88,26 @@ impl StableApiDefinition for Definition {
         ptr
     }
 
+    #[inline]
+    unsafe fn rbasic_class(&self, obj: VALUE) -> VALUE {
+        let rbasic = obj as *const crate::RBasic;
+
+        (*rbasic).klass
+    }
+
+    #[inline]
+    unsafe fn rbasic_frozen_p(&self, obj: VALUE) -> bool {
+        // FIXME: Why this does not work?
+        // if self.special_const_p(obj) {
+        //     true
+        // } else {
+        //     let rbasic = obj as *const crate::RBasic;
+        //     ((*rbasic).flags & crate::ruby_value_type::FL_FREEZE as VALUE) != 0
```

Comment:
isn't it:

```suggestion
        //     ((*rbasic).flags & crate::ruby_fl_type::FL_FREEZE as VALUE) != 0
```

Reply:
Yup yup, typo. ruby_fl_type is not available I had to whitelist it in the bindings configuration to make it work for the bignum_p PRs.